command/create_test.go

```diff
@@ -30,8 +30,8 @@ func TestCreateIssueAsExpected(t *testing.T) {
 
 	var inputStruct = struct {
 		Project   string `json:"project"`
-		IssueType string `json:"issueType"`
-		Title     string `json:"title"`
+		IssueType string `json:"issuetype"`
+		Summary   string `json:"summary"`
 	}{"FLYTE", "Story", "test story"}
 	input := toJson(inputStruct, t)
```

Comment:
Can we ditch the anonymous structure and `toJson` and instead pass everything as bytes to the function? Similar to the other tests. It might be nice to have since you're already working on this heh.

I don't see the benefit of creating a structure, filling it up, serializing it only to have it deserialized later, seems a bit counterintuitive to me personally. 🤔